.github/workflows/prerelease_linux.yml

```diff
@@ -9,13 +9,27 @@ on:
 
 env:
   TAG: ${{ github.event.release.tag_name }}
+  GPG_MAIL: 'infrastructure-eng@newrelic.com'
```

Comment:
I would recommend create a secret for this

Reply:
Done! thanks for the suggestion :bow: